employee_seniority_level/security/hr_employee_seniority_level.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.model.access" id="hr_employee_seniority_level_access_user">
```

Comment:
Why do we need this file if there's already an `ir.model.access.csv` ? :confused:

Reply:
Indeed, left over from previous implementation ?!
Removed